lib/dynamo.js

```diff
@@ -11,18 +12,15 @@ const AWS = require('aws-sdk');
  * ddbClient.getAsync(..etc).then(...etc);
  *
  */
-module.exports = class DynamoDB {
+module.exports = {
+  getInstance: (config) => {
+    if (cache.get('awsInitialized') !== true) {
+      throw new Error('AWS has not been initialized yet, please check config & code');
+    }
 
-  constructor(config) {
-    this.config = config;
-    this.client = new AWS.DynamoDB.DocumentClient(config);
-    Promise.promisifyAll(this.client);
-  }
-
-  // getRawClient() {
-  //   const rawClient = new AWS.DynamoDB(config);
-  //   Promise.promisifyAll(rawClient);
-  //   return rawClient;
-  // }
+    this._instance = this._instance || new AWS.DynamoDB.DocumentClient(config);
```

Comment:
If we knew what the config here was going to be, I guess we could set this on the cache Map too instead of this._instance ... future improvement maybe?